episodes/11/assets/network/subnets.yaml

```diff
@@ -3,7 +3,7 @@ kind: Subnet
 metadata:
   name: sample-subnet1
 spec:
-  cidrBlock: 192.168.64.0/18
+  cidrBlock: 10.0.64.0/18
```

Comment:
192.168.0.0/16 is the configured Pod CIDR, if we use it here, then we risk communication issues between the Pods and resources within the VPC

Reply:
We actually should be okay here because for this demo the CAPI cluster is being deployed to a separate VPC entirely. It will access the RDS instance over public internet 👍